Assets/Scripts/Interactables/Collectable.cs

```diff
@@ -18,5 +28,23 @@ private void OnTriggerEnter2D(Collider2D other)
                 Destroy(gameObject);
             }
         }
+
+        /// <summary>
+        /// Chooses a random sprite from the possible sprites of this collectable.
+        /// Should only be run in editor.
+        /// </summary>
+        public void RandomizeSprite()
+        {
+            spriteRenderer.sprite = possibleSprites[Random.Range(0, possibleSprites.Length)];;
```

Comment:
nit: I created a `SelectRandom` method in https://github.com/AdenC123/MtStringmore/blob/main/Assets/Scripts/Util/RandomUtil.cs, so you could probably just call `possibleSprites.SelectRandom()`